src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/actionfurtherevidence/ActionFurtherEvidenceMidEventHandlerTest.java

```diff
@@ -153,60 +153,6 @@ public void givenAPostponementRequestInOtherThanInterlocTcwAction_thenAddAnError
                 is(ActionFurtherEvidenceMidEventHandler.POSTPONEMENTS_REVIEWED_BY_TCW));
     }
 
-    @Test
-    public void givenAPostponementRequestWithGaps_thenAddAnError() {
```

Comment:
@jaberelfasi Why is this test removed please ? Looks valid to me based on the requirement.